Global Date/Time/Number Formats: Add API Descriptions (#8697)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/GlobalConfig/dateTimeFormat.md

---
id: GlobalConfig.dateTimeFormat
type: LocalizationFormat | Record
default: undefined
---
---
##### shortDescription
Specifies the default date and time format for all DevExtreme components in the application.

---

> Assign a string value to this property to use a [predefined format](/Documentation/ApiReference/Common_Types/#Format). Specify a [custom format string](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) or a [format function](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Function) to use a custom format.

Reviewer: ```suggestion
This property can accept a [predefined format string](/Documentation/ApiReference/Common_Types/#Format). Specify a [custom format string](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) or a [format function](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Function).
```

---

##### jQuery

    <!-- tab: index.js -->
    DevExpress.config({
        dateTimeFormat: 'yyyy-MM-dd, HH:mm:ss',
    });

##### Angular

    <!-- tab: app.component.ts -->
    import config from "devextreme/core/config";

    config({
        dateTimeFormat: 'yyyy-MM-dd, HH:mm:ss',
    });

##### Vue

    <!-- tab: App.vue -->
    import config from "devextreme/core/config";

    config({
        dateTimeFormat: 'yyyy-MM-dd, HH:mm:ss',
    });

##### React

    <!-- tab: App.tsx -->
    import config from "devextreme/core/config";

    config({
        dateTimeFormat: 'yyyy-MM-dd, HH:mm:ss',
    });

---

> You can configure default formats for specific locales. To do this, assign a key-value pair object to this property. Use locale identifiers as keys to define formats for specific locales. Use the `default` key to specify a format for all other locales:

Reviewer: ```suggestion
You can configure default formats for specific locales. To do this, assign a key-value pair object to this property. Use locale identifiers as keys. Use the `default` key to specify a format for all other locales:
```
